src/util/graph.creator.ts

```diff
@@ -84,12 +84,35 @@ export class GraphCreator {
         };
     }
 
-    private static resolveModule(rawImportLocation: string): string {
+    private static resolveModule(rawImportLocation: string, originDirectory: string): string {
         if (/[.]ts$/.test(rawImportLocation)) {
+            // import ends in .ts => has to be a file
             return rawImportLocation;
         }
+
+        if (
+            fs.existsSync(path.join(originDirectory, rawImportLocation, '.ts')) &&
+            fs.lstatSync(path.join(originDirectory, rawImportLocation, '.ts')).isFile()
```

Comment:
What does `lstatSync` return for non-existent paths? undefined? 
If so, we could maybe save ourselves the existsSync-check

Reply:
It throws. We could replace existsSync with a catch block. Don't know if that is desireable.

Reply:
Ah, in that case, let's keep it this way, thanks :+1:
Not sure if it would help in terms of readability, but might make sense to have two methods like `isValidDirectory`/`isValidFile` which do the fs-stuff, but for the two-three usages that we have here, it's probably a bit whatever